App crashes on Start on Samsung S5

Caused by: android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
        at android.app.ContextImpl.startActivity(ContextImpl.java:738)
        at android.app.ContextImpl.startActivity(ContextImpl.java:725)
        at android.content.ContextWrapper.startActivity(ContextWrapper.java:345)
        at io.horizontalsystems.bankwallet.modules.pin.PinActivity$Companion.startForUnlock(PinActivity.kt:243)
        at io.horizontalsystems.bankwallet.modules.pin.PinModule.startForUnlock(PinModule.kt:56)
        at io.horizontalsystems.bankwallet.core.managers.LockManager.lock(LockManager.kt:44)
        at io.horizontalsystems.bankwallet.core.managers.LockManager.willEnterForeground(LockManager.kt:39)
        at io.horizontalsystems.bankwallet.core.managers.BackgroundManager.onActivityStarted(BackgroundManager.kt:24)

--- a/app/src/main/java/io/horizontalsystems/bankwallet/modules/pin/PinActivity.kt
+++ b/app/src/main/java/io/horizontalsystems/bankwallet/modules/pin/PinActivity.kt
@@ -239,7 +239,7 @@ class PinActivity : BaseActivity(), NumPadItemsAdapter.Listener, FingerprintAuth
         fun startForUnlock() {
             val intent = Intent(App.instance, PinActivity::class.java)
             intent.putExtra(keyInteractionType, PinInteractionType.UNLOCK)
-            intent.flags = Intent.FLAG_ACTIVITY_EXCLUDE_FROM_RECENTS and Intent.FLAG_ACTIVITY_NEW_TASK and Intent.FLAG_ACTIVITY_CLEAR_TOP
+            intent.flags = Intent.FLAG_ACTIVITY_EXCLUDE_FROM_RECENTS or Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TOP
             App.instance.startActivity(intent)
         }
     }

